Converting Audience Sync Troubleshooting section and details dropdowns into FAQ section with H3s (#9298)
braze-inc/braze-docs: _docs/_partners/canvas_steps/google_audience_sync.md

| *Exited Canvas* | Number of users who have exited the Canvas. This occurs when the last step in a Canvas is a Google step. |
{: .reset-td-br-1 .reset-td-br-2 role="presentation" }

## Frequently asked questions

### Why can I not select multiple fields to match in my Google Audience Step configuration?

Google Customer Match has strict requirements around how these audiences are formatted and what customer information is included. Specifically, mobile advertiser IDs need to be uploaded separately from customer contact information (such as email and phone number). For more details, refer to [Google's Customer Match documentation](https://support.google.com/google-ads/answer/7659867?hl=en#undefined).

### How long will it take for my audiences to sync in Google?

It can take anywhere between 6 to 12 hours for an audience to be synced into Google. 

### I've synced an audience, so why is the audience size in Google zero?

For privacy purposes, the user list size will show zero until the list has at least 1,000 members. After that, the size will be rounded to the two most significant digits.

### I've synced an audience into Google, but my ads are not serving.

> Check that your audiences contain at least 5,000 users to ensure that ads start serving.

Reviewer: ```suggestion
Check that your audiences contain at least 5,000 users so that ads can start serving.
```
